dali/base/dautils.cpp

```diff
@@ -1912,6 +1912,17 @@ IRemoteConnection *getSortedElements( const char *basexpath,
 //==================================================================================
 
 #define PAGE_CACHE_TIMEOUT (1000*60*10)
+static unsigned pageCacheTimeoutMilliSeconds = PAGE_CACHE_TIMEOUT;
+void setPageCacheTimeoutMilliSeconds(unsigned timeoutSeconds)
+{
+    pageCacheTimeoutMilliSeconds = 1000 * timeoutSeconds;
+}
+
+static unsigned maxPageCacheItems = 0; //no limit
```

Comment:
We should add a default limit, even if moderately high, perhaps 1000.
Should also default in xsd to be same.